packages/site-admin/src/components/page/index.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+import { NavigableRegion } from '../../interface';
+/**
+ * Internal dependencies
+ */
+import Header from './header';
+import './style.scss';
+/**
+ * Types
+ */
+import type { PageProps } from './types';
+import type { PropsWithChildren } from 'react';
+
+export function Page( {
+	title,
+	subTitle,
+	actions,
+	children,
+	className,
+	hideTitleFromUI = false,
+}: PropsWithChildren< PageProps > ) {
+	const classes = clsx( 'a8c-site-admin-page', className );
+
+	return (
+		<NavigableRegion className={ classes } ariaLabel={ title }>
+			<div className="a8c-site-admin-page__content">
```

Comment:
Is the `div.a8c-site-admin-page__content` necessary, or can it be "merged" into the `NavigableRegion` component ?

Reply:
I’m not entirely sure, to be honest.
As we mentioned at this stage, we mostly aim to copy and paste the core implementation to move quickly.

I think we realize we could eventually get rid of it, but maybe that’s something for the next stage of the project.
What do you think?